### DISCUSSION TITLE
Add Link to Docs on the Demo site

### DISCUSSION THREAD

**Discussion by arndom** [gilbarbara/react-joyride#967] [Ideas]
## Feature Request

- A link to the docs for the demo site: https://react-joyride.com/

  - The link can only found in the GitHub readme

## Possible Solution

- The Link could be placed in the examples' footer:

<img width="708" alt="image" src="https://github.com/gilbarbara/react-joyride/assets/40564798/a8bbdd74-06da-432e-a8d6-b45e3cf31757">

**Comment by alexisruiz96** [comment]
This could be great! Actually i was searching it in the demo but it's not included.

**Comment by gilbarbara** [comment]
Done!
